packages/wonder-stuff-sentry/src/stringify-nested-contexts.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import {safeStringify} from "@khanacademy/wonder-stuff-core";
+import type {SentryContexts} from "./types.js";
+
+export const stringifyNestedContexts = (
+    contexts: SentryContexts,
+): SentryContexts => {
+    const stringifiedContexts = {};
+
+    // For each context,
+    for (const [key, value] of Object.entries(contexts)) {
+        stringifiedContexts[key] = {};
+
+        // For each top-level property of a context (each context is an object).
+        // $FlowIgnore[incompatible-call]
+        for (const [subKey, subValue] of Object.entries(value)) {
+            if (typeof subValue === "object") {
+                if (Array.isArray(subValue)) {
+                    stringifiedContexts[key][subKey] = subValue.map((v) =>
+                        typeof v === "object" ? safeStringify(v) : v,
```

Comment:
I'm sure you're already aware, but just checking that you want to be handling `null` in cases like this (as `typeof null === "object"` is `true`).

Reply:
Thanks for checking. I wasn't sure what I wanted to do with `null`, to be honest. Perhaps I should be more specific to make it clear that we really mean it.